vuepress/docs/next/tutorials/solution/install-standard-demo.md

```diff
@@ -5,47 +5,74 @@ sidebarDepth: 2
 # Entando Standard Banking Demo
 
 This tutorial will guide you through installing a demo application using the Local Hub and a set of Entando
-bundles. This solution template includes: 
+Bundles. This solution template includes: 
 
-- microservices
-- micro frontends
-- multiple pages
+- Microservices
+- Micro frontends
+- Multiple pages
 - CMS content
 
-The goal of this exercise is to demonstrate how Entando bundles can be used to: 
+The goal of this exercise is to demonstrate how Entando Bundles can be used to: 
 
-- quickly install and create functionality in an Entando Application
-- enable packaged business capabilities
-- allow developers to reuse full stack operations via bundles
+- Quickly install and create functionality in an Entando Application
+- Enable packaged business capabilities (PBCs)
+- Allow developers to reuse full-stack operations via bundles
 
-Some of the key elements of the template are reviewed in the [Application Details section](#application-details) below.
+Some of the key elements of this template are reviewed in the [Application Details section](#application-details) below.
 
 ## Installation
 
-There are numerous assets installed as part of the Standard Banking Demo. Entando Bundles can include more or less components, depending on objectives. It is recommended that organizations develop guidelines for bundle sizing that fit the goals of their applications and teams.
+Numerous assets installed in the course of the Standard Banking Demo. Entando Bundles can include more or less components, depending on objectives. It is recommended that organizations develop guidelines for bundle sizing that fit the goals of their applications and teams.
```

Comment:
more or less / objectives applies to entando bundles in general; instead of "the" or no "the" let's identify who has the objectives - user or developer. i think developer works best here...